storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -4223,7 +4247,7 @@ void xtrabackup_backup_func(void) {
 
   recv_is_making_a_backup = true;
   bool data_copying_error = false;
-
+  std::shared_ptr<xb::dd_tablespaces> xb_dd_space;
```

Comment:
btw, I dont see concurrent usages of dd_space. You probably can use std::unique_ptr

Reply:
backup function create the concurrent threads based on the number of paralles.

Reply:
I see. Makes sense to keep.